examples/pretty-colored.rs

```diff
@@ -0,0 +1,85 @@
+//! This example shows how to configure fern to output really nicely colored logs
+//!  - when the log level is error, the whole line is red
+//!  - when the log level is warn, the whole line is yellow
+//!  - when the log level is info, the level name is green and the rest of the line is white
+//!  - when the log level is debug, the whole line is white
+//!  - when the log level is trace, the whole line is gray ("bright black")
+
+extern crate chrono;
+extern crate fern;
+#[macro_use]
+extern crate log;
+
+use fern::colors::{Color, ColoredLevelConfig};
+
+fn main() {
+    set_up_logging();
+    // let's simulate some logging
+    info!("starting simulation!");
+    for i in 0..26 {
+        trace!("loading: {}%, very verbose debbuging information", 4*i);
```

Comment:
Note: `4*i` will be reformatted to `4 * i` with rustfmt- I'll run this after merging.